eng/scripts/Invoke-GenerateAndBuildV2.ps1

```diff
@@ -123,14 +126,28 @@ if ($relatedTypeSpecProjectFolder) {
           })
         } else {
             $relativeSdkPath = Resolve-Path $sdkProjectFolder -Relative
-            GeneratePackage `
-            -projectFolder $sdkProjectFolder `
-            -sdkRootPath $sdkPath `
-            -path $relativeSdkPath `
-            -downloadUrlPrefix $downloadUrlPrefix `
-            -serviceType "data-plane" `
-            -generatedSDKPackages $generatedSDKPackages `
-            -specRepoRoot $swaggerDir
+            if ($sdkProjectFolders -contains $sdkProjectFolder) {
+              # Existed SDK project case, needs to generate code
+              GeneratePackage `
+              -projectFolder $sdkProjectFolder `
+              -sdkRootPath $sdkPath `
+              -path $relativeSdkPath `
+              -downloadUrlPrefix $downloadUrlPrefix `
+              -serviceType "data-plane" `
+              -generatedSDKPackages $generatedSDKPackages `
+              -specRepoRoot $swaggerDir
+            } else {
+              # New SDK project case, code is already generated by emitter. So, skip code generation
```

Comment:
For new sdk project case, other steps within `GeneratePackage` still need to be called.

Reply:
It wouldn't re-generate as the `skipGenerate` flag is set when call `GeneratePackage` in line#147.